app/templates/administrative-units/administrative-unit/sites/site/index.hbs

```diff
@@ -31,6 +31,12 @@
               <:label>Type vestiging</:label>
               <:content>{{@model.site.siteType.label}}</:content>
             </Item>
+            {{#if this.isAndereVestigen}}
+              <Item>
+                <:label>Name</:label>
+                <:content>{{@model.site.siteTypeName}}</:content>
+              </Item>
+            {{/if}}
```

Comment:
**Nitpick:** do we want to display this label even if there is no value for name? I think that optional fields without a value are generally not displayed in this frontend.